engine/src/main/java/org/terasology/engine/subsystem/headless/HeadlessGraphics.java

```diff
@@ -31,9 +31,7 @@
 import org.terasology.engine.subsystem.headless.renderer.HeadlessRenderingSubsystemFactory;
 import org.terasology.engine.subsystem.headless.renderer.ShaderManagerHeadless;
 import org.terasology.rendering.ShaderManager;
-import org.terasology.rendering.assets.animation.MeshAnimation;
-import org.terasology.rendering.assets.animation.MeshAnimationData;
-import org.terasology.rendering.assets.animation.MeshAnimationImpl;
+import org.terasology.rendering.assets.animation.*;
```

Comment:
No start imports 😉

Reply:
No prob. I assume the IntelliJ default settings just got me there.